airgun/views/http_proxy.py

```diff
@@ -0,0 +1,81 @@
+from widgetastic.widget import (
+    Text,
+    TextInput,
+    View,
+)
+from widgetastic_patternfly import BreadCrumb
+
+from airgun.views.common import (
+    BaseLoggedInView,
+    SatTab,
+    SearchableViewMixin,
+)
+from airgun.widgets import (
+    MultiSelect,
+    SatTable,
+)
+
+
+class HTTPProxyView(BaseLoggedInView, SearchableViewMixin):
+    title = Text("//h1[text()='HTTP Proxies']")
+    new = Text("//a[contains(@href, 'http_proxies/new')]")
+    table = SatTable(
+        './/table',
+        column_widgets={
+            'Name': Text('./a'),
+            'URL': Text('./a'),
```

Comment:
'./a' is this valid? Do both have same locator?

Reply:
yes, it's valid.